src/strategies/discord.ts

```diff
@@ -4,17 +4,23 @@ import {
   OAuth2StrategyVerifyCallback,
 } from "./oauth2";
 
+export type DiscordScope = "activities.read" | "activities.write" | "applications.builds.read" | "applications.builds.upload" |
```

Comment:
Could you add a comment with the link from where this came from? So I could check them to see if they are up to date in the future.

Reply:
Done.
The link is also listed in the scope input for the strategy options, but having it in both places surely doesn't hurt.